docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,164 @@
+
+Introduction
+************
+
+The core mission of edX is to expand access to education for everyone.  We expect that any user interfaces that are developed for the Open edX platform can be used by everyone, regardless of any physical limitation they may have.  The Open edX platform is used everyday by people who may not be able to see, hear or have the dexterity to use traditional modes of computer interaction (like the mouse).  Understanding a few core concepts about how people with disabilities use the web and web applications should make the guidance in this document a little easier to understand.
+
+* Someone who cannot see cannot use a mouse (nor can those with neurological conditions that prevent the kind of precise hand eye coordination required to use a mouse).
+* Many people with disabilities use "`Assistive Technology <http://www.w3.org/TR/WCAG20/#atdef>`_" (like a screen reader) to interact with their computer, web browser, and web applications.
+* Anything conveyed visually (like an element's role, state or other properties) must also be conveyed programmatically so Assistive Technologies know the element's role, what state it's currently in and how users can interact with it.  *Protip:* This information often comes for free when you use native HTML5 elements.
+* Many people with mobility impairments use custom keyboards or keyboard emulators.
+* Some people rely on speech to interact with web applications and need to be able to address interactive elements.
+* Someone who cannot hear or hear well requires visible text captions for any audible content or an equivalent visual cue.
+* Users may customize their operating system, browser or web page display properties to make web applications easier for them to use (like reversing contrast, increasing font size, or removing images). 
+
+These are just a few core concepts developers need to keep in mind when developing user interfaces that work for everyone.  More information is available from the W3C's Web Accessibility Initiative's `How People with Disabilities Use the Web: Overview <http://www.w3.org/WAI/intro/people-use-web/Overview.html>`_.
+
+Make Your Images Accessible
+************************
+
+All images require a `text alternative <http://www.w3.org/TR/WCAG20/#text-altdef>`_ with the exception of any image that is purely decorative or has a text alternative adjacent to it.  Regardless of whether or not an image element requires a text alternative, *all* ``<img>`` elements require an ``alt`` attribute, even if the value of that attribute is ``""`` (NULL).  Because an author error should not prevent a screen reader user from using a web site, screen readers will expose the path/to/the/image as a last resort, in an effort to provide some useful information to the user regarding the purpose of the image.  If your image is purely decorative, or has a text equivalent immediately adjacent to it, use a NULL alt attribute i.e. ``alt=""``.
```

Comment:
I don't know...  technically, its a NULL value.  That is how its always referenced in technical documents.